scripts/release/git_add_gitignore_bypass_release_files.sh

```diff
@@ -0,0 +1,23 @@
+#!/bin/bash
+set -euxo pipefail
+
+# Runs `git add -f --all X` for every entry X from list_gitignore_bypass_release_files.sh.
+# This is used to bypass the .gitignore to commit files that should ONLY be committed for releases.
+
+# Change to the directory where this script is located
+cd "$(dirname "$0")"
+
+# Get the list of files from list_gitignore_bypass_release_files.sh
+gitignore_bypass_release_files=$(./list_gitignore_bypass_release_files.sh)
+
+# Run `git add` from root directory
+cd ../..
+
+while IFS= read -r line; do
+    # Only proceed if the line is not empty
+    if [[ -n "$line" ]]; then
+        # If `git add` doesn't match any files, it errors.
+        # Swallow error. Release technician should detect issue.
```

Comment:
Sorry, but this comment confused me.
How will the release technician detect an issue if the error is swallowed?

I guess what you are telling me is that this runs before or after some test
that ensures all the critical files are present;
that test's error is not swallowed.

Reply:
Yes. See MCM instructions: https://code.amazon.com/packages/CryptoolsScripts/blobs/209dcb9dd7dc2d46356828fb257f23538435a5bb/--/templates/mcm/release_common/preflight_changelog.txt#L91-L99